test/memberstatus_assertion.go

```diff
@@ -71,6 +72,29 @@ func (a *MemberStatusAssertion) HasHostConditionErrorMsg(expected string) *Membe
 	return a
 }
 
+type AddUsage func(map[string]int)
+
+func OfNodeRole(role string, usage int) AddUsage {
+	return func(nodesUsage map[string]int) {
+		nodesUsage[role] = usage
+	}
+}
+
+func (a *MemberStatusAssertion) HasMemoryUsage(usages ...AddUsage) *MemberStatusAssertion {
```

Comment:
could be renamed to `HasMemoryUsages` (plural) but that's a minor detail

Reply:
I thought that the word `usage` should be used as an uncountable word in this context :man_shrugging:

Reply:
yeah, on second thought, you're probably right ;)